src/main/java/ca/on/oicr/pineryreports/reports/impl/OctaneCountsReport.java

```diff
@@ -168,24 +159,7 @@ public String getTitle() {
         + (userIds.isEmpty() ? " for all users" : " for selected users");
   }
   
-  private static final String ATTR_CATEGORY = "Sample Category";
-  private static final String ATTR_TISSUE_TYPE = "Tissue Type";
-  private static final String ATTR_TISSUE_ORIGIN = "Tissue Origin";
-  private static final String ATTR_SLIDES = "Slides";
-  private static final String ATTR_DISCARDS = "Discards";
-  private static final String ATTR_CONSUMED = "Slides Consumed";
-  private static final String ATTR_REMAINING = "Remaining";
-  private static final String ATTR_STAIN = "Stain";
-  
-  private static final String SAMPLE_CLASS_SLIDE = "Slide";
-  private static final String SAMPLE_CLASS_WHOLE_RNA = "whole RNA";
-  
-  private static final String SAMPLE_CATEGORY_IDENTITY = "Identity";
-  private static final String SAMPLE_CATEGORY_TISSUE = "Tissue";
-  private static final String SAMPLE_CATEGORY_STOCK = "Stock";
-  private static final String SAMPLE_CATEGORY_ALIQUOT = "Aliquot";
 
-  private static final String STAIN_HE = "Hematoxylin+Eosin";
   
   @Override
   protected void collectData(PineryClient pinery) throws HttpResponseException {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 39 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)